### PR TITLE
Story/configure travis

### PR DESCRIPTION
**Pull request by jicardu**
## Summary

- Add .travis.yml
- Add database.travis.yml
- Add badge of build status
- Add RSpec
- Add dotenv

## Trello Card

https://trello.com/c/iEfVjFEg/12-configurar-travisci

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@
 /yarn-error.log
 
 .byebug_history
+.env
```
Comment (sebasdeldi) [inline .gitignore]: EOF

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  - bundle exec rspec spec -fd
+  - bundle exec rubocop app spec --format simple
+
+#notifications:
+#  email:
+#    recipients:
+#      - rails-bootstrap-dev@wolox.com.ar
+
+cache:
+  bundler: true
```
Comment (sebasdeldi) [inline .travis.yml]: EOF

File: Gemfile [hunk 3]
```diff
@@ -40,6 +40,8 @@ group :development, :test do
   gem 'capybara', '~> 2.13.0'
   gem 'selenium-webdriver'
   gem 'rubocop', '~> 0.75.0', require: false
+  gem 'dotenv-rails'
```
Comment (sebasdeldi) [inline Gemfile]: I think that using [pessimistic lock](https://thoughtbot.com/blog/rubys-pessimistic-operator) on your gems versions is a good idea, It's a little extra point you add to the stability of your dependencies within the project.

File: config/database.travis.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+test:
+  adapter: postgresql
+  encoding: unicode
+  database: travis_wbooks_api_test
+  username: travis_wbooks_api
+  password: travis_wbooks_api
```
Comment (sebasdeldi) [inline config/database.travis.yml]: EOF

File: config/database.yml [hunk 5]
```diff
@@ -24,6 +24,8 @@ default: &default
 development:
   <<: *default
   database: wbooks_api
+  username: wbooks_api
```
Comment (sebasdeldi) [inline config/database.yml]: Consider using env variables here too.
Any idea why you should do it ?

File: config/database.yml [hunk 6]
```diff
@@ -57,7 +59,9 @@ development:
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: wbooks_api
+  database: wbooks_api_test
+  username: wbooks_api
```
Comment (sebasdeldi) [inline config/database.yml]: same

File: Gemfile (hunk 3 above)
Comment (jicardu) [reply Gemfile]: Ok :)

**Pull request merged**